source/assets/public/css/nav.css

```diff
@@ -0,0 +1,132 @@
+/* Navigation */
+.nav__room {
+  padding-top: 6rem;
+}
+
+nav {
```

Comment:
As below in the JS, I disagree with scoping these styles to the element, we may legitimately want to use `nav` elements in other places

Reply:
What about using the ID selector: `#menu`, and then we leave `nav` selectors for general purpose styling. (e.g. scoping lists and anchor styling within `<nav>` elements)

Reply:
Sure